compiler/rustc_session/src/config/cfg.rs

```diff
@@ -176,6 +176,8 @@ pub(crate) fn default_configuration(sess: &Session) -> Cfg {
     // NOTE: These insertions should be kept in sync with
     // `CheckCfg::fill_well_known` below.
 
+    ins_none!(sym::autodiff_fallback);
```

Comment:
This is make the `autodiff_fallback` cfg stably available, which should at least be gated behind nightly (with tests, doc, ...), but is your goal to even let users use `autodiff_fallback`?